.github/workflows/e2e-test.yml

```diff
@@ -31,12 +39,13 @@ jobs:
     - name: Run BMO e2e Tests
       env:
         BMC_PROTOCOL: ${{ inputs.bmc-protocol }}
+        GINKGO_FOCUS: "${{ inputs.ginkgo-focus }}"
       # We need a new shell to pick up the new group. That is why we do the sudo -s -u $USER ...
       # Remove the pre-installed go version. We install the exact version we need.
       run: |
         sudo usermod -a -G libvirt $USER
         sudo rm /usr/bin/go
-        sudo -s -u $USER bash ${{ github.workspace }}/hack/ci-e2e.sh
+        sudo -s -u $USER --preserve-env bash ${{ github.workspace }}/hack/ci-e2e.sh
```

Comment:
Note: This was a mistake in the previous PR. The env vars were not passed through. By adding --preserve-env we make sure that the environment variables really are available in the script.